composer.json

```
@@ -24,14 +24,15 @@
 		"source": "https://github.com/infinum/eightshift-boilerplate"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.7",
-		"infinum/eightshift-coding-standards": "^1.6",
-		"php-stubs/wordpress-stubs": "^6.1.1",
-		"szepeviktor/phpstan-wordpress": "^1.1.7"
+		"dealerdirect/phpcodesniffer-composer-installer": "^v1.0.0",
+		"infinum/eightshift-coding-standards": "2.0.0-beta",
+		"php-stubs/wordpress-stubs": "^6.3",
+		"szepeviktor/phpstan-wordpress": "^1.1.7",
+		"wp-cli/wp-cli": "^2.9"
 	},
 	"require": {
-		"php": "^7.4 || <8.0",
-		"infinum/eightshift-libs": "^6.0.0",
+		"php": "^7.4 || >=8.0",
```

```suggestion
		"php": ">=7.4",
```
we can always change it to >=7.4 <= 8.2 for example if 8.3 adds breaking changes
(copy-paste from the FE libs comment)